repos/system_upgrade/el7toel8/actors/cupsfiltersmigrate/actor.py

```diff
@@ -0,0 +1,86 @@
+from leapp.actors import Actor
+from leapp.libraries.common.reporting import report_generic
+from leapp.models import Report, CupsFiltersModel
+from leapp.tags import ApplicationsPhaseTag, IPUWorkflowTag
+
+"""
+cups-browsed configuration directory
+"""
+
+CONFDIR = '/etc/cups/'
+
+"""
+Configuration file name
+"""
+
+config = 'cups-browsed.conf'
+
+"""
+Directives to be added
+"""
+
+directives = [
+              'LocalQueueNamingRemoteCUPS RemoteName', 
+              'CreateIPPPrinterQueues All'
+             ]
```

Comment:
What do you think if we move CONFDIR, config and directives to inside the functions they are used. They seem to be used one single time and being close to where they are used helps on debuging, imo.